src/main/java/de/tum/in/www1/artemis/service/ParticipationService.java

```diff
@@ -1284,6 +1284,15 @@ public StudentParticipation findOneWithEagerResultsAndCourse(Long participationI
         return studentParticipationRepository.findOneByIdWithEagerResultsAndExerciseAndEagerCourse(participationId);
     }
 
+    /**
+     *
```

Comment:
This comment is a bit pointless (maybe even incomplete if results are returned, too)